roles/openshift_manage_node/tasks/main.yml

```diff
@@ -34,5 +34,18 @@
   when: "'nodename' in openshift.node"
   delegate_to: "{{ openshift_master_host }}"
 
+- name: Wait for node to be ready
```

Comment:
Nodes are not required to be ready before they can start.  We should not be adding this, because you should be able to install an openshift cluster where no sdn plugin has been installed (and hence nodes won't be ready).